thyrel-web/src/components/room/draw/DrawColorPicker.tsx

```diff
@@ -55,7 +56,7 @@ export default function DrawColorPicker({
         <Box
           key={index}
           display="flex"
-          flexDirection="row"
+          flexDirection={flexDirection === 'column' ? 'row' : 'column'}
```

Comment:
```suggestion
          flexDirection={flexDirection}
```
tu peux directement met flexDirection comme ça ;)

Reply:
Non, parce que quand flexDirection est une colonne celui-ci doit être un row et vice versa

Reply:
ah merde ta raison, c'est un peu contre inuitif mais si ça un sens ok